shepherd/api/app.py

```diff
@@ -49,7 +49,7 @@ def create_app(debug=None) -> web.Application:
     :return: a new application object
     """
 
-    app = web.Application(debug=debug if debug is not None else os.getenv('DEBUG', False))
+    app = web.Application(debug=debug if debug is not None else os.getenv('DEBUG', False), client_max_size=10*1024**3)
```

Comment:
@Teyras in which projects contains aiohttp? This seems to be useful.
@blazekadam btw this might be the problem with large payloads (demotool I guess?)

Reply:
@petrbel yes, this must be it, the default value is `1024**2` bytes ~= 1 MB which is quite easy to exceed.